frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.module.scss

```diff
@@ -2,6 +2,7 @@
 
 button.trigger {
   flex-shrink: 0;
+  display: flex;
```

Comment:
This was previously removed in
https://github.com/mozilla/fx-private-relay/commit/92ec5a7781341347028cddd2d68d8925a1421a07 and then forgotten to
restore in https://github.com/mozilla/fx-private-relay/commit/73844bc721445d697576aa85365cc940e4c2c9e2. Not a biggie,
but explains why I had to make this change :)